### PR TITLE
Changes unlocking to use ast.literal_eval

### LINKED ISSUE #211: OK unlocking should not eval
I've seen several students type `5/2` as an answer to a `print(5/2)` WWPD, even though this is not the correct answer.

Instead, we could try to parse and match the syntax trees, or something similar, to handle cases like
`[1,2,3] ~ [1, 2, 3]`

### PR DESCRIPTION
**Pull request by jathak**
Resolves #211.

### CODE REVIEW

File: tests/protocols/unlock_test.py [hunk 1]
```diff
@@ -205,23 +203,27 @@ def testMultipleChoice_multipleFailsBeforeSuccess(self):
             else:
                 self.validateRecord(attempt, answer=self.SHORT_ANSWER, correct=True)
 
+    EVAL_ANSWER = ['[1, 2, 3, 4]']
+    CORRECT_EVAL = ['[1,2,3,2+2]']
+    INCORRECT_EVALS = ['[4,3,2,1]', '{1,2,3,4}', '[1,2,3,8/2]']
```
Comment (Sumukh) [inline tests/protocols/unlock_test.py]: For future reference: `ast.literal_eval` supports addition/subtraction of integers (but not other operations/other types)

I don't think this is worth a comment in the test.

Also might be worth a shot to give it something it just fail on - like `raise Exception("test")` (which throws a SyntaxError)

**Comment by Sumukh** [comment]
Great. 

Before merge: 
- [ ] Test an unlocking suite from the lab. 
- [ ] Runs test with something that produces a syntax error (like `raise Exception` or `a = 1`)

File: tests/protocols/unlock_test.py (hunk 1 above)
Comment (jathak) [inline tests/protocols/unlock_test.py]: I think the 8/2 in the third incorrect eval should cause an error.

**Comment by jathak** [comment]
Tested with lab02 and it seems to work as expected.

**Comment by Sumukh** [comment]
Thanks for testing it. Merging now.

**Pull request merged**